client/project_policy.go

```diff
@@ -28,8 +28,8 @@ type PolicyUpdatePayload struct {
 	SkipRedundantDeployments    bool   `json:"skipRedundantDeployments"`
 	RunPullRequestPlanDefault   bool   `json:"runPullRequestPlanDefault"`
 	ContinuousDeploymentDefault bool   `json:"continuousDeploymentDefault"`
-	MaxTtl                      string `json:"maxTtl"`
-	DefaultTtl                  string `json:"defaultTtl"`
+	MaxTtl                      string `json:"maxTtl,omitempty"`
```

Comment:
If empty string. Don't send a json with an empty string. Instead don't send the field.
This should be equivalent to "null".